docs/Pipelines/PET_Introduction.md

```diff
@@ -35,6 +35,25 @@ bids
       └─ sub-CLNC001_ses-M000_trc-18FFDG_pet.nii.gz
 ```
 
+## Reconstruction methods
+
+PET data can be reconstructed into the spatial distribution of the injected radio-tracer through various methods.
+The specification of the method used to reconstruct a given image is supported by the [BIDS standard](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/09-positron-emission-tomography.html#pet-recording-data).
+
+There exists four reserved values:
+
+- `acdyn`: for reconstructions with attenuation correction of dynamic data
+- `acstat`: for reconstructions with attenuation correction of static data
+- `nacdyn`: for reconstructions without attenuation correction of dynamic data
+- `nacstat`: for reconstructions without attenuation correction of static data
+
+In addition to these values, when working with ADNI data, the following values are possible:
+
+- `coregdyn`: Corresponds to "ADNI processing steps 1"
```

Comment:
What do you call ADNI processing steps ?

Reply:
I have added a link to the ADNI documentation for PET images which explains this (see commit [d523bd8](https://github.com/aramis-lab/clinica/pull/1300/commits/d523bd80d15dcabff1b42d30a9e2b073034b7740)).